### ISSUE TITLE
Spanish Translation Final Tasks!

### ISSUE THREAD

**Issue by tesseralis** [reactjs/es.reactjs.org#158]
¡Felicidades!

Y'all are amazing! 🎊It's been two weeks and you're translated the **entire React documentation** into Spanish! Give yourselves a big round of applause 👏

Now, there are couple more errands to do before we can finally deploy:

* [ ] Replace "reactjs.org" to "es.reactjs.org in the following places:
  * [ ] package.json
  * [ ] site-constants.js
  * [ ] gatsby-config.js
  * [ ] MarkdownPage (the link href in "Edit this page")
* [ ] Change `html.js` to have the attribute `lang="es"`
* [ ] Deploy the site, either locally or through Netlify/Heroku, and go over each section to:
  * [ ] Make sure there are no formatting errors (dangling markdown/html tags) or untranslated English
  * [ ] Make sure the sections and style are consistent.

Again, amazing work, and I can't wait to introduce es.reactjs.org to the world!

**Comment by alejandronanez** [comment]
It’s happening!

**Comment by alejandronanez** [comment]
Thanks to @loconluis for #159!

**Comment by alejandronanez** [comment]
I'll review the whole site during the weekend (life happens)! 😸

**Comment by carburo** [comment]
The site is deployed on [Netlify](https://es-react-js.netlify.com/) so it's easier for everyone to take a look and review it.

**Comment by devrchancay** [comment]
It looks great on Netlify but they need to review #155

**Comment by Darking360** [comment]
Greetings y'all 🚀 awesome! I'll take a look maybe tomorrow or Monday and see all 💪

**Comment by tesseralis** [comment]
Would you all be okay with the current netlify deployment becoming es.reactjs.org? If so, we can get @ericnakagawa or @bvaughn to point es.reactjs.org to it.

**Comment by tesseralis** [comment]
I looked over it and had a few thoughts:

The headings in the home page are styled strangely?
<img width="764" alt="screen shot 2019-02-16 at 6 17 43 pm" src="https://user-images.githubusercontent.com/1278991/52907512-e5a5dd80-3217-11e9-87e0-9c3e66562939.png">
<img width="763" alt="screen shot 2019-02-16 at 6 17 49 pm" src="https://user-images.githubusercontent.com/1278991/52907513-e6d70a80-3217-11e9-886e-688813fdd5b7.png">

We should at least translate the nav bar:

<img width="372" alt="screen shot 2019-02-16 at 6 19 52 pm" src="https://user-images.githubusercontent.com/1278991/52907517-f5252680-3217-11e9-84ed-96fc1df8305c.png">

And the page title:

<img width="188" alt="screen shot 2019-02-16 at 6 17 34 pm" src="https://user-images.githubusercontent.com/1278991/52907519-00785200-3218-11e9-9dcb-31f5efc317d0.png">

**Comment by carburo** [comment]
@tesseralis There is a pending PR (#160) that should translate the navbar. I should have it reviewed and merged later tonight.

**Comment by tesseralis** [comment]
@carburo great!

_Elementos DOM_/DOM Elements is inconsistent: https://es-react-js.netlify.com/docs/dom-elements.html

**Comment by tesseralis** [comment]
PR for the home page title: https://github.com/reactjs/es.reactjs.org/pull/163

**Comment by tesseralis** [comment]
This is less important but we can also search for links to en.wikipedia.org or developer.mozilla.org  and seeing if there are any English links that have a better Spanish version.

**Comment by tesseralis** [comment]
One more thing: the text in the Search bar is cut off
<img width="189" alt="screen shot 2019-02-17 at 1 02 57 pm" src="https://user-images.githubusercontent.com/1278991/52919186-85b14480-32b4-11e9-93f0-65cd2214d87c.png">

Maybe it's best to just shorten it to "Buscar"?

**Comment by alejandronanez** [comment]
Hey @tesseralis @carburo @Darking360 @dmoralesm!

I just did a quick run through the whole website and everything seems to be ok 🎉 .

I found some minor things that could be improved and opened the respective PRs #169 #170 #171

**Comment by Darking360** [comment]
[PR #170](https://github.com/reactjs/es.reactjs.org/pull/170) merged! 🚀 

I'll take a look into core 💪

**Comment by Darking360** [comment]
Hey guys 💪 I was looking into the translation so far, I opened [this PR](https://github.com/reactjs/es.reactjs.org/pull/174) in order to add my feedback and some things that I observed so far, I'm on `React.Component` right now, but I wanted you to take a look into what's there so far to not double fix the same thing. 

I've seen some accents left, un-translated parts, code in spanish, some links to english docs and internal links not working for bad naming, so I've been fixing those, I hope to finish by tomorrow at noon 🤔 

**CC:** @alejandronanez @carburo @dmoralesm @tesseralis

**Comment by lex111** [comment]
I would also like to join in the congratulations to the Spanish translation team. ¡Un gran trabajo, todos! :clap: 

@tesseralis excuse me for writing here, but since these are final tasks, I’m wondering why there’sn’t a task to set up the search, as I understand it, need to change the [Algolia API key](https://github.com/reactjs/es.reactjs.org/blob/ed07fa53dc9ab7950e630900f7da3a7c821d63a6/src/components/LayoutHeader/DocSearch.js#L25)?

**Comment by tesseralis** [comment]
@lex111 you're totally right! We uh... should fix that.

**Comment by lex111** [comment]
@tesseralis it would be nice to have instructions on how to do this :+1:

**Comment by tesseralis** [comment]
@lex111 I... have absolutely no idea.

Maybe @gaearon @ericnakagawa @bvaughn could help?

**Comment by tesseralis** [comment]
Seems like the site uses Algolia [docsearch](
https://community.algolia.com/docsearch/), which is invitation only. So we can probably do a few things:

* Have one of the maintainers apply for it pointing to es.reactjs.org. I have no idea if they’re quick at responding to requests, and it doesn’t seem like a good long term strategy for our 30+ languages
* disable the search for now (there should be a var in the search component) so people don’t see confusing English results
* think about switching search providers? Is that something possible? There are a few alternatives on the gatsby site.

**Comment by tesseralis** [comment]
Eh, I’m sure it’ll be fine with Algolia. The jp.vuejs.org has it working. I’ll just have to add it to the “starting out” guide

**Comment by escorponox** [comment]
I saw some minor issues in hooks-effect in the last review. I'll send a PR asap.

**Comment by tesseralis** [comment]
Guess what: es.reactjs.org

**Comment by bvaughn** [comment]
I'm trying to figure out how to force redirect HTTP -> HTTPS, but for now at least the master branch of this repo is being built by Netlify and I've setup the subdomain to point to the result.

**Comment by tesseralis** [comment]
IIRC you have to wait until the dns propagates and then they’ll either do it automatically or there’s a big green button you can push.

**Comment by tesseralis** [comment]
Just applied for Algolia docsearch. Should get a response in a few days. Until then, would we want to disable the search bar?

**Comment by bvaughn** [comment]
Yeah, thanks Nat! I was hoping that was the case based on [this Stack Overflow post](https://stackoverflow.com/questions/49712714/can-netlify-redirect-traffic-from-http-to-https-without-forcing-ssl/49712715#49712715). Just waiting to see.

> Until then, would we want to disable the search bar?

This might be good, so long as there's an issue to re-enable later.

**Comment by sergiodxa** [comment]
I saw the docs are already deployed, but it's not using HTTPS, Netlify should give a SSL cert for free with just a click in the admin panel.

**Comment by bvaughn** [comment]
> I saw the docs are already deployed, but it's not using HTTPS

HTTPS is also available. It's just not forcefully redirecting. Supposedly this is something we can make Netlify do, I'm just not sure how yet.
